### PR TITLE
interfaces: updates to login-session-observe, network-manager and modem-manager interfaces

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**
Some updates detected while working on NM based on 20.04 release.

### CODE REVIEW

File: interfaces/builtin/login_session_observe.go [hunk 1]
```diff
@@ -77,6 +77,27 @@ dbus (receive)
...
+    member={Session,User,Seat}Removed
+    peer=(label=unconfined),
+
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/login1
+    interface=org.freedesktop.login1.Manager
+    member=PrepareFor{Shutdow,Sleep}
+    peer=(label=unconfined),
+
```
Comment (jdstrand) [inline interfaces/builtin/login_session_observe.go:100]: These are all fine.

File: interfaces/builtin/modem_manager.go [hunk 2]
```diff
@@ -67,7 +67,7 @@ capability sys_admin,
 
 # For {mbim,qmi}-proxy
 unix (bind, listen) type=stream addr="@{mbim,qmi}-proxy",
-/sys/devices/**/usb**/{descriptors,manufacturer,product} r,
+/sys/devices/**/usb**/{descriptors,manufacturer,product,bInterfaceClass,bInterfaceSubClass,bInterfaceProtocol,bInterfaceNumber} r,
```
Comment (jdstrand) [inline interfaces/builtin/modem_manager.go:70]: This is fine.

File: interfaces/builtin/network_manager.go [hunk 3]
```diff
@@ -92,12 +92,9 @@ network packet,
...
 # Allow access to configuration files generated on the fly
-# from netplan and let NetworkManager store its DHCP leases
-# in the dhcp subdirectory so that console-conf can access
-# it.
-/run/NetworkManager/ w,
-/run/NetworkManager/{,**} r,
-/run/NetworkManager/dhcp/{,**} w,
+# from netplan and let NetworkManager store its configuration
+# in the same place.
+/run/NetworkManager/{,**} wr,
```
Comment (jdstrand) [inline interfaces/builtin/network_manager.go:97]: Nitpick: please use 'rw'. That said, can you comment on why we need blanket 'w' for the whole dir? I think it is fine, but would like to at least have it documented in the PR (and really, the commit message) on why it was done.

File: interfaces/builtin/network_manager.go [hunk 4]
```diff
@@ -145,7 +142,7 @@ dbus (send)
      bus=system
      path="/org/freedesktop/resolve1"
      interface="org.freedesktop.resolve1.Manager"
-     member="SetLink{DNS,Domains}"
+     member="SetLink{DNS,MulticastDNS,Domains,LLMNR}"
```
Comment (jdstrand) [inline interfaces/builtin/network_manager.go:145]: Looks fine.

**Comment by alfonsosanchezbeato** [comment]
@jdstrand thanks for the review! I have update the comment in the commit that changes permissions to /run/NetworkManager, and changed it to "rw".

**Pull request merged**